status-im/status-react
# Feature/discover empty images #1999

fixes #1999 

### Summary:
Nice screens shown when there are no discoveries yet.

### Steps to test:
- Open Status
- Create a fresh account (so that discoveries would still be empty)
- Open Discover
- See the empty screens
- Open sidebar, create a status with a hashtag
- See the empty screens replaced with content

[comment]: # (PRs will only be accepted if squashed into single commit.)
status: ready
